apps/native/app/src/utils/lifecycle/setup-notifications.ts

```diff
@@ -2,23 +2,21 @@ import messaging, {
   FirebaseMessagingTypes,
 } from '@react-native-firebase/messaging'
 import {
-  addNotificationReceivedListener,
-  addNotificationResponseReceivedListener,
   DEFAULT_ACTION_IDENTIFIER,
   Notification,
   NotificationResponse,
-  setNotificationHandler,
 } from 'expo-notifications'
 import { navigateTo, navigateToNotification } from '../../lib/deep-linking'
-import { isIos } from '../devices'
 
 export const ACTION_IDENTIFIER_NO_OPERATION = 'NOOP'
 
 export async function handleNotificationResponse({
   actionIdentifier,
   notification,
 }: NotificationResponse) {
-  const link = notification.request.content.data?.link
+  const link =
+    notification.request.content.data?.link ??
+    notification.request.content.data?.clickActionUrl
```

Comment:
If the clickActionUrl exist in the notification data then we use that for deeplinking

Reply:
Can we have both link and clickActionUrl? Then it will use the link